ArduCopter/defines.h

```diff
@@ -81,6 +81,7 @@ enum aux_sw_func {
 #define UNDEFINED_FRAME 0
 #define MULTICOPTER_FRAME 1
 #define HELI_FRAME 2
+#define ROT_FRAME 2
```

Comment:
If you make this the same value has the heli frame then this isn't needed

Reply:
ok for ROT_FRAME it should have been set to 3
OK for your contribution rules, I will apply them. Do I submit a new PR or shall I wait for you to reject this one first?
The EKF filter is clearly diverging, I will post a video to show it.
The first commit will be removed but I still don't get it, this i2c magnetometer is clearly not working with the aero board, is that possible to get it more documented so that I know how to connect it?